Userland/Libraries/LibJS/Tests/error-stack.js

```diff
@@ -0,0 +1,26 @@
+test("Anonymous function", function () {
+    let stackString = (() => {
+        return Error();
+    })().stack;
+    let [header, ...stackFrames] = stackString.split("\n");
+
+    expect(header).toBe("Error");
+    expect(!!stackFrames[0].match(/^    at Error \(.*\.js:\d+:\d+\)$/)).toBeTrue();
+    expect(!!stackFrames[1].match(/^    at .*\.js:\d+:\d+$/)).toBeTrue();
```

Comment:
Can we hardcode the file name (not path) and line numbers at least?

Reply:
I omitted the column numbers, because they seem to point to the end of the word, or the first parenthesis, instead of the beginning of the word, although specifically accessing `source_range.start`